src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor

```diff
@@ -4,7 +4,25 @@
 <div @ref="RootElement"
      style="@StyleBuilder.Value"
      class="@ClassBuilder.Value">
-    <span class="flexContainer">
+    <span>
+        <i class="bit-icon bit-icon--@MessageBarIconName"></i>
+    </span>
+    <span class="text-container @( (Truncated && !ElementTruncateState) || IsMultiline ? "not-truncate" : "")">
```

Comment:
there is no need to have the "text-container" class in here.

Reply:
in here we have 3 spans one for left side icon one for right side icons and the other one which is in thee middle is for text I thought it's better that we assign a class to the one containing our text

Reply:
yes, you are right, but as we spoke about these issues before, because the structure of the component is clear and constant for us (the developers) we can use the ":nth-child()" selector here.

Reply:
Fixed